--- jail.local	2023-06-13 13:34:45.418568293 +0200
+++ jail.local	2023-06-13 13:34:31.720464090 +0200
@@ -205,8 +205,8 @@
 # iptables-multiport, shorewall, etc) It is used to define
 # action_* variables. Can be overridden globally or per
 # section within jail.local file
-banaction = iptables-multiport
-banaction_allports = iptables-allports
+banaction = firewallcmd-rich-rules[actiontype=]
+banaction_allports = firewallcmd-rich-rules[actiontype=]
 
 # The simplest action to take: ban only
 action_ = %(banaction)s[port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
@@ -282,9 +282,14 @@
 # normal (default), ddos, extra or aggressive (combines all).
 # See "tests/files/logs/sshd" or "filter.d/sshd.conf" for usage example and details.
 #mode   = normal
+
+enabled = true
 port    = ssh
 logpath = %(sshd_log)s
 backend = %(sshd_backend)s
+bantime = 10m
+findtime = 10m
+maxretry = 5
 
 
 [dropbear]
