--- sshd_config	2023-06-17 17:05:10.306520679 +0200
+++ sshd_config 2023-06-17 17:04:04.352088647 +0200
@@ -31,7 +31,7 @@
 # Authentication:
 
 #LoginGraceTime 2m
-#PermitRootLogin prohibit-password
+PermitRootLogin no
 #StrictModes yes
 #MaxAuthTries 6
 #MaxSessions 10
@@ -55,7 +55,7 @@
 #IgnoreRhosts yes
 
 # To disable tunneled clear text passwords, change to no here!
-#PasswordAuthentication yes
+PasswordAuthentication no
 #PermitEmptyPasswords no
 
 # Change to yes to enable challenge-response passwords (beware issues with
@@ -107,7 +107,7 @@
 #VersionAddendum none
 
 # no default banner path
-#Banner none
+Banner /etc/banner
 
 # Allow client to pass locale environment variables
 AcceptEnv LANG LC_*
